### PR TITLE
chore: [#187578669] disable auto tests in content-repo on commits

### PR DESCRIPTION
**Pull request by aadedejifearless**
### Ticket

This pull request resolves [#187578669](https://www.pivotaltracker.com/story/show/187578669).

### Approach

- Update `config.yml` by:
  - Disabling the auto test for content on commits
  -  Adding a scheduled job to run the `build and test` suit against the `content-repo` daily

## Code author checklist

- [X] I have rebased this branch from the latest main branch
- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation on the engineering documentation website
- [X] I have not used any relative imports
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config
- [X] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [X] I have checked for and removed instances of unused config from CMS
- [X] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [X] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -934,6 +934,7 @@ workflows:
                 - /^webflow-fundings-sync-.*/
                 - /^webflow-licenses-sync-.*/
                 - /^webflow-industries-sync-.*/
+                - /^content-repo-.*/
```
Comment (jphechter) [inline .circleci/config.yml:937]: [boulder] Right now this is matching looking for a string to contain this substring`content-repo-`. This wouldn't catch the branch name.

This should be looking for an exact match.

Comment applies to all instances.
```suggestion
                - /^content-repo$/
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -1205,3 +1210,102 @@ workflows:
...
+          filters:
+            branches:
+              only:
+                - content-repo
+    jobs:
+      - docker-build-builder-image:
+          filters:
+            branches:
+              only:
+                - content-repo
```
[the remark this answers is not in the document]
Comment (aadedejifearless) [reply .circleci/config.yml:1227]: You are absolutely correct. I wasnt too sure about the behavior here. The schedule trigger forces the jobs the jobs to stay only in the branch. I'll make the change.

File: .circleci/config.yml [hunk 3]
```diff
@@ -1205,3 +1210,102 @@ workflows:
                 - accessibility-testing-repo
     jobs:
       - deploy-accessibility-testing
+
+  scheduled-daily-tests-content-repo:
```
Comment (jphechter) [inline .circleci/config.yml:1214]: [pebble] This is effectively duplicating the `build-and-test` workflow. Is there a DRY way to approach this?

Comment (aadedejifearless) [reply .circleci/config.yml:1214]: Responded to this in Slack. The short answer to this is the code is currently DRY. We are only referencing the jobs already created.

**Review by jphechter — APPROVED** [review]

**Pull request merged**